src/sagemaker/jumpstart/utils.py

```diff
@@ -402,6 +402,12 @@ def verify_model_region_and_return_specs(
             f"JumpStart model ID '{model_id}' and version '{version}' " "does not support training."
         )
 
+    if model_specs.eula_model:
+        LOGGER.info(
```

Comment:
See prior comment, we can perhaps give them the ability to retrieve the EULA from our public bucket